src/mlpack/methods/ann/activation_functions/softplus_function.hpp

```diff
@@ -0,0 +1,114 @@
+/**
+ * @file softplus_function.hpp
+ * @author Vivek Pal
+ *
+ * Definition and implementation of the softplus function.
+ *
```

Comment:
The required fields of INPROCEEDINGS are: author, title, booktitle, year so, we should make sure there is a booktitle.